Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2015 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+.PHONY: all format clean master manager fuzzer executor
+
+all: master manager fuzzer executor
+
```

Comment:
also copy ./manager/example.cfg to ./manager.cfg, so that system is ready to go after make
there must a cp flag that does not overwrite existing files (and don't fail at the same time)
or you want it to go to some special dir, so that it does not show up in 'git status'? if so, we need to figure out where to put config files (master will probably also switch to config file)